docs/application-connector/docs/06-01-application.md

```diff
@@ -11,7 +11,7 @@ kubectl get crd applications.applicationconnector.kyma-project.io -o yaml
 
 ## Sample custom resource
 
-This is a sample resource that registers the `re-prod` App which offers one service.
+This is a sample resource that registers the `system_prod` App which offers one service.
```

Comment:
CR name cannot have `_` in it's name, so there is also a mistake in the definition below. It would be nice to correct it also. I would suggest using `-`.

Reply:
Fixed